<?xml version="1.0"?>
<!--
  Author : Jaeyong Park
  This is for the test in the simulation environment
  It will run tracking mpc, along with target pid controller.
 -->
<launch>
  <arg name="robot_namespace" default="scout"/>
  <group ns="$(arg robot_namespace)">
    <!-- Track -->
    <node name="catching_mpc_controller" pkg="drone_catching_mpc" type="catching_mpc_controller" respawn="false" clear_params="true" output="screen">
      <rosparam file="$(find drone_catching_mpc)/config/gazebo_test.yaml" />
    </node>
    <!-- Taregt -->
    <node name="target_controller" pkg="drone_catching_mpc" type="target_controller" respawn="false" clear_params="true" output="screen">
      <!-- <rosparam file="$(find drone_catching_mpc)/config/gazebo_test.yaml" /> -->
    </node>
  </group>

</launch>
